salt/modules/capirca_acl.py

```diff
@@ -556,8 +556,8 @@ def _get_policy_object(
     for filter_ in filters:
         if not filter_ or not isinstance(filter_, dict):
             continue  # go to the next filter
-        filter_name = filter_.keys()[0]
-        filter_config = filter_.values()[0]
+        filter_name = next(iter(filter_.keys()))
+        filter_config = next(iter(filter_.values()))
```

Comment:
`filter_name, filter_config = next(iter(filter_.items()))`

Reply:
haha I should have seen that. Thanks, will fix